dummy/src/toga_dummy/test_implementation.py

```diff
@@ -84,9 +84,11 @@ def _extract_classes(self):
             if isinstance(node, ast.ClassDef):
                 if self.is_required_for_platform(node):
                     self._classes[node.name] = node  # use the class name as the key
-            elif isinstance(node, ast.Assign):
-                # Allow a class with no new methods to be defined by assigning from an
-                # existing class.
+            elif isinstance(node, ast.Assign) and node.col_offset == 0:
+                # Allow a class with no new methods to be defined by assigning
+                # from an existing class. The col_offset means we only pay
+                # attention to assignments at the top level of a module, not
+                # assignments inside method bodies.
```

Comment:
This line was catching the `old_value = self._get_value('value')` call in the dummy implementation. The line is required to allow for `MainWindow = Window` as a top level module assignment to be sufficient to define the existence of an interface class. This isn't a great check - but as soon as we're at 100% testbed coverage, we're not going to need the implementation checks anyway, so I figured a bandaid solution was sufficient.